src/Ledger/Foreign/HSLedger/BaseTypes.agda

```diff
@@ -159,59 +159,59 @@ instance
         ; maxBlockSize = to maxBlockSize
         ; maxTxSize = to maxTxSize
         ; maxHeaderSize = to maxHeaderSize
-        ; maxValSize = to maxValSize
-        ; minUTxOValue = to minUTxOValue
-        ; poolDeposit = to poolDeposit
         ; keyDeposit = to keyDeposit
+        ; poolDeposit = to poolDeposit
         ; Emax = to Emax
         ; nopt = to nopt
         ; pv = to pv
+        ; minUTxOValue = to minUTxOValue
```

Comment:
Although this has been retired after the `Mary` era and the Haskell implementation doesn't have it anymore, I opted to move it here to preserve consistency with the `Shelley` era field order, rather than moving it to the end of the record.